frontend/src/components/customer/ProfileDetailsCard.tsx/ProfileDetailsCard.tsx

```diff
@@ -0,0 +1,480 @@
+                                                                }}
+                                                            />
+                                                        </LocalizationProvider>
+                                                    )}
+                                                />
+                                            </Grid>
+                                            <Grid item>
+                                                <TextField
+                                                    id="outlined-basic"
+                                                    label="Postal Code"
+                                                    variant="outlined"
+                                                    InputLabelProps={{
+                                                        sx: {
+                                                            fontFamily: "Lato",
+                                                            fontSize: "1rem",
+                                                            fontWeight: "400",
+                                                        },
+                                                    }}
+                                                    {...register(
+                                                        "postal_code",
+                                                        {
+                                                            required:
+                                                                "Postal Code is required",
+                                                        }
+                                                    )}
+                                                    error={!!errors.postal_code}
+                                                    helperText={
+                                                        errors.postal_code
+                                                            ?.message
+                                                    }
+                                                    FormHelperTextProps={{
+                                                        sx: {
+                                                            fontSize: "1rem",
+                                                        },
+                                                    }}
+                                                />
+                                            </Grid>
+                                        </Grid>
+                                    </Grid>
+                                    <Grid item xs={5.8}>
+                                        <Grid
+                                            container
+                                            direction="column"
+                                            gap={3}
+                                            justifyContent="flex-start"
+                                            alignItems="flex-start"
```

Comment:
What happens if I remove these? Does it really change everything?
Rmb to not declare properties that are by default already assigned by MUI